src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -547,6 +587,32 @@ private SnapshotSourceModel makeSourceModelFromSource(SnapshotSource source) {
         return sourceModel;
     }
 
+    private SnapshotAccessInfoModel makeSnapshotAccessInfoModelFromSnapshot(Snapshot snapshot) {
+        SnapshotAccessInfoModel accessInfoModel = new SnapshotAccessInfoModel();
+
+        // Currently, only BigQuery is supported.  Parquet specific information will be added here
+        accessInfoModel.bigQuery(new SnapshotAccessInfoModelBigQuery()
+            .datasetName(snapshot.getName())
+            .projectId(snapshot.getProjectResource().getGoogleProjectId())
+            .link(new ST(BIGQUERY_BASE_LINK)
```

Comment:
`ST` seems like a heavyweight solution to a problem that can be solved using `String.format()`. Is there a reason why `ST` is needed here?

Reply:
Yeah, I found that it makes the code easier to read.  I've run into so many bugs before with format where you need to know the position of the replacement and you end up getting the wrong one.  This seems like it makes it more obvious what's going on (particularly since the string pattern definition is at the top of the class)